### PR TITLE
feat: use new OPENAI_API_KEY variable in the docker part

### PR DESCRIPTION
**Pull request by eric-glb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,17 +205,19 @@ Use the provided `Dockerfile` to build a container:
...
+           --env OPENAI_API_KEY \
            --volume gpt-config:/home/app/.config/shell-gpt \
            --volume gpt-cache:/tmp/shell_gpt \
        sgpt --chat rainbow "what are the colors of a rainbow"
 ```
 
 Example using an alias:
 ```shell
-alias sgpt="docker run --rm --volume gpt-config:/home/app/.config/shell-gpt --volume gpt-cache:/tmp/shell_gpt sgpt"
+alias sgpt="docker run --rm --env OPENAI_API_KEY --volume gpt-config:/home/app/.config/shell-gpt --volume gpt-cache:/tmp/shell_gpt sgpt"
```
Comment (TheR1D) [inline README.md:219]: Can't we pass it during run? We could omit config volume in this case.
```
docker run  --rm -e OPENAI_API_KEY="your api key" --volume gpt-cache:/tmp/shell_gpt sgpt
```

Comment (eric-glb) [reply README.md:219]: Yep, I'll split the example

Comment (eric-glb) [reply README.md]: Oh, I misunderstood. I'll add also this.

Comment (TheR1D) [reply README.md]: I think we can keep one option with env variable.
Remove config volume from readme.
Remove config volume from Dockerfile, for example:
```
FROM python:3-slim

ENV PYTHONDONTWRITEBYTECODE 1
ENV PYTHONUNBUFFERED 1
ENV PIP_ROOT_USER_ACTION ignore
WORKDIR /app
COPY . /app

RUN pip install --no-cache --upgrade pip \
 && pip install --no-cache /app \
 && addgroup --system app && adduser --system --group app \
 && mkdir -p /tmp/shell_gpt \
 && chown -R app:app /home/app \
                     /tmp/shell_gpt

USER app

VOLUME /tmp/shell_gpt

ENTRYPOINT ["sgpt"]
```

Comment (eric-glb) [reply README.md]: #62  does not remove the command asking for the API key if not found, does it?
Or do you plan to remove this, therefore removing the config volume in advance?

Comment (TheR1D) [reply README.md]: It doesn't remove it, users will be still prompted for the api key. But with docker, we can force users to use ENV variable only, I think it makes it easy to use.

Comment (eric-glb) [reply README.md]: Done

**Comment by TheR1D** [comment]
I just made few small changes: removed extra spaces, and inline comment `# where to store your chat history`. I think it is clear from description above why we need it.

**Pull request merged**